### DISCUSSION TITLE
FGLair Integration: Continue with Custom Integration or merge into Home Assistant Core?

### DISCUSSION THREAD

**Discussion by bigmoby** [bigmoby/fglair_for_homeassistant#173] [Polls]
With the release of Home Assistant 2024.9, the official integration for FGLair has been added. However, the current version of FGLair in the core release doesn't seem complete or fully functional to me. I would like to open a discussion on how we should proceed with our project.

My point of view:
The advantage (PRO) of continuing with a custom integration is that any further developments or bug fixes could be more agile, as they wouldn't be tied to the official Home Assistant release schedule. However, the downside (CONS) is that, at the moment, I am practically the sole maintainer of this project, so all the work essentially depends on me. If we decide to merge the custom integration into the core version of Home Assistant, there is a possibility that the base of contributors working on the project could expand — although there’s also a chance that this might not happen!

**Comment by SgtFlippy** [comment]
I feel I'm not up to speed about all this to vote. Just very happy my AC's are available in HA.
Not sure what would change if I changed over, wanted to try but can't find my password again 😅

I do like the quick solutions we often get from you, but I very much understand you won't be able to keep this up forever on your own. So I'm a bit in the middle on it all.

**Reply by bigmoby** [reply]
Hi @SgtFlippy 
Thank you so much for your contribution!
I believe the current implementation I created is fully functional, except for some potential bugs reported by users with air conditioners that are either not easily integrated or have certain limitations. 
Anyway...I don't think I will be adding any more features... the air conditioners work, and that's it! 😅

**Reply by SgtFlippy** [reply]
If it does what it says on the tin, who are we to complain? 🤣

**Reply by bigmoby** [reply]
No no, sorry, maybe I expressed myself badly! 😅 Surely there is or will be some bug, but I don't think I will add other product features...which there wouldn't be because the air conditioners are currently managed at the maximum of their features as in the official Fujitsu mobile app.

**Comment by chiefcomm** [comment]
@bigmoby on my side I haven't widely tested the new integration more than it fails with "This device is not supported by FujitsuHVAC" which leaves me to want to retain the current custom integration which has always worked flawlessly :smile:

**Reply by bigmoby** [reply]
Hi @chiefcomm 🙌
Yes, also my ac "is not supported" 😅...ehm! ...so persevere at moment! 👍
